fastify.js

```diff
@@ -93,7 +131,7 @@ function build () {
       const handle = node[req.method]
 
       if (handle && req.method === 'GET') {
-        handleNode(handle, params, req, res, null)
+        handleNode(handle, params, req, res, urlUtil.parse(req.url, true).query)
```

Comment:
I prefer the second option, considering that managing querystring is not trivial, probably is better do a different pr for that, and in this one focus on the body validation.